File: Seminar/src/main/java/org/sopt/Seminar/domain/member/exception/MemberNotFoundException.java
package org.sopt.Seminar.domain.member.exception;

import org.sopt.Seminar.global.common.exception.ErrorType.NotFound;
import org.sopt.Seminar.global.common.exception.SoptException;

public class MemberNotFoundException extends SoptException.NotFoundException {
    public MemberNotFoundException() {
        super(NotFound.USER_NOT_FOUND, "해당 멤버가 존재하지 않습니다.");
    }
}

File: kopring/week2/kotlin-semina/src/main/kotlin/org/sopt/kotlinsemina/domain/member/exception/MemberNotFoundException.kt
package org.sopt.kotlinsemina.domain.member.exception

import org.sopt.kotlinsemina.global.common.exception.ErrorType.NotFound.*
import org.sopt.kotlinsemina.global.common.exception.SoptException

class MemberNotFoundException: SoptException.NotFoundException(MEMBER_NOT_FOUND, "해당 멤버가 존재하지 않습니다.")